### PR TITLE
added assignment 1 files

### PR DESCRIPTION
**Pull request by lizaku**
Add Q1 and Q2 of the first assignment

### CODE REVIEW

File: lizaku/assignment1/Q1.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        w1 = w1.lower()
+        w2 = w2.lower()
+    if len(w1) != len(w2):
+        return False
+    letters1 = list(w1)
+    letters2 = list(w2)
+    while letters1 != [] or letters2 != []:
+        for letter in letters1:
+            if letter in letters2:
+                letters1.remove(letter)
```
Comment (ariava) [inline lizaku/assignment1/Q1.py]: Please explain in a comment why you are removing letters from the list after finding them :)

File: lizaku/assignment1/Q1.py [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    words2 = [x for x in re.split('[' + string.punctuation + '\\s]', s2) if x]
+    if len(words1) != len(words2):
+        return False
+    while words1 != [] or words2 != []:
+        for w1 in words1:
+            found = False
+            for w2 in words2:
+                if anagrams_words(w1, w2):
+                    found = True
+                    words1.remove(w1)
```
Comment (ariava) [inline lizaku/assignment1/Q1.py]: Same as above, please explain in a comment why you are removing words after finding them.

File: lizaku/assignment1/Q1.py [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+                    words2.remove(w2)
+                    break
+            if not found:
```
Comment (ariava) [inline lizaku/assignment1/Q1.py]: I think you can use the "else" clause of the for loop here.
http://book.pythontips.com/en/latest/for_-_else.html for more info.

**Review by ariava — CHANGES_REQUESTED** [review]
Very cool, good job! Please have a look at my comments.

File: lizaku/assignment1/Q1.py (hunk 3 above)
Comment (lizaku) [reply lizaku/assignment1/Q1.py]: I didn't know about it! Thanks!

File: lizaku/assignment1/Q1.py (hunk 1 above)
Comment (lizaku) [reply lizaku/assignment1/Q1.py]: @ariava I explained, but this is comment is pending... What am I to do now?

**Comment by lizaku** [comment]
@ariava what else do I need to do?

**Review by ariava — APPROVED** [review]
Looks great, thanks!

**Comment by ariava** [comment]
Please merge the branch as soon as you like! Looks awesome!

**Comment by lizaku** [comment]
Thank you!

**Pull request merged**